<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkDock</class>
 <widget class="QWidget" name="MarkDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="noMarks">
         <property name="text">
          <string>No marks</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="markDeadlocks">
         <property name="text">
          <string>Mark deadlocks</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="markStates">
         <property name="text">
          <string>Mark states</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="markTransitions">
         <property name="text">
          <string>Mark transitions</string>
         </property>
         <attribute name="buttonGroup">
          <string>buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Mark states</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QComboBox" name="clusterMatchStyle">
            <item>
             <property name="text">
              <string>Mark cluster if any state is marked</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mark cluster if all states are marked</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="stateMatchStyle">
            <item>
             <property name="text">
              <string>Match any of the following</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Match all of the following</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Match the following separately</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="markRuleList"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="add">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remove">
              <property name="text">
               <string>Remove</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Mark transitions</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QListWidget" name="markedActionList"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
